presto-spi/src/main/java/com/facebook/presto/spi/eventlistener/QueryStatistics.java

```diff
@@ -62,14 +66,18 @@ public QueryStatistics(
             long totalRows,
             long outputBytes,
             long outputRows,
-            long writtenBytes,
-            long writtenRows,
+            long logicalWrittenBytes,
```

Comment:
Document (comment) what's "logical bytes" and how it differs from "physical".
For example:
- i guess "logical" is before any compression. Am i right?
- i guess "logical" is before "unrolling" RLE or Dictionary blocks. Am i right?

Reply:
Not sure it belongs here, would be better off in QueryStats.

```
    @JsonProperty
    public DataSize getLogicalWrittenDataSize()
    {
        return succinctBytes(
                operatorSummaries.stream()
                        .filter(stats -> stats.getOperatorType().equals(TableWriterOperator.class.getSimpleName()))
                        .mapToLong(stats -> stats.getInputDataSize().toBytes())
                        .sum());
    }
```